README.md

```diff
@@ -70,10 +70,24 @@ To make a debug build (to include symbols into elf file, turn off optimizations,
 ### OpenOCD
 Flashing the MMC microcontroller via SWD/JTAG is supported for CMSIS-DAP and Jlink compatible probes through OpenOCD. You can specify the debug probe with the flag `-DDEBUG_PROBE=<probe_name>`, valid options are `cmsis-dap` (default), `jlink`, `digilent_jtag_hs3` and `xvc` (support is not merged to the official OpenOCD source yet, so you build our patched version: https://github.com/lnls-dig/openocd/tree/fix-afcv3-flashing).
 
+You can specify the debug probe in the CMake configuration with the -DDEBUG_PROBE=<probe_name> flag. Example for CMSIS-DAP:
+
 	cmake ~/openmmc/ -DBOARD=afc -DVERSION=3.1 -DDEBUG_PROBE=cmsis-dap
 
 This will create a `openocd.cfg` file in `<build_dir>/out`.
 
+In scenarios where the AFC board is in M7 (Error State) or M1 (Shutdown State), the Xilinx Virtual Cable (XVC) interface is particularly useful, allowing firmware upgrades over Ethernet via JTAG. This enables recovery without physical access to the crate.
+
+To configure a build using XVC:
+
+    cmake ~/openmmc/ -DBOARD=afc -DVERSION=4.0 -DDEBUG_PROBE=xvc -DXVC_HOST=de-23rabpm-co-cratectrl -DXVC_PORT=2542
```

Comment:
Prefer not using real our internal hostname / port here. Change it to `-DXVC_HOST=xvc_hostname -DXVC_PORT=xvc_port_number`